src/localLighthouse.js

```diff
@@ -160,11 +176,20 @@ export default async ({
       overrides,
       throttling,
       throttlingMethod,
-      url
-    });
+      url,
+      verbose
+    };
 
-    auditResults.push(lighthouseAuditResult);
-    index++;
+    if (options.emulatedFormFactor != 'both') {
```

Comment:
Can we do a strict comparison for consistency (`!==`)? Sorry the linting doesn't catch this.

Reply:
Can we update the `README.md` file to add the `both` option?

```diff
- oneOf(['mobile', 'desktop'])
+ oneOf(['both', 'mobile', 'desktop'])
```

Reply:
Can we delete `lighthouseCaller` and instead just add the logging within the if / else blocks. And you could set the default option to `mobile` updating this line to `emulatedFormFactor = 'mobile'`:

https://github.com/foo-software/lighthouse-check/blob/78a9b0a60128d0764fc73dd77028773a49414fd9/src/lighthouseCheck.js#L20

Reply:
Yes, i was waiting for your approval of source code before adding it to documentation.

Reply:
> Can we delete `lighthouseCaller` and instead just add the logging within the if / else blocks. And you could set the default option to `mobile` updating this line to `emulatedFormFactor = 'mobile'`:
> 
> https://github.com/foo-software/lighthouse-check/blob/78a9b0a60128d0764fc73dd77028773a49414fd9/src/lighthouseCheck.js#L20
I added lighthouse caller for a more DRY approach (IMO) instead of having multiple prints , i will update default emulatedFormFactor